Lib/pdb.py

```diff
@@ -1411,6 +1414,7 @@ def do_break(self, arg, temporary=False):
     # To be overridden in derived debuggers
     def defaultFile(self):
         """Produce a reasonable default."""
+        assert self.curframe is not None
```

Comment:
This was meant to be added in `completedefault` not here. This function must check that `curframe` exists.